ParabolInc/parabol
# feat: Add team filter in timeline history page

# Description
Added team filter in timeline history page

## Demo
![2023-07-16 11 07 05](https://github.com/ParabolInc/parabol/assets/1879975/cc6907d9-b918-4929-ac9d-f9cc4a8b9326)

## Testing scenarios

- [ ] Team filter in history page
  - Go to history page
  - Select a team from the team filter dropdown menu
  - Observe the events are filtered for that team
  - Verify infinity scrolling still work

- [ ] Combination of team filter & event type filter
  - Go to history page
  - Select a team from the team filter dropdown menu, and then select an event type from the event filter dropdown menu
  - Verify the events are filtered
  - Verify you can "reset" by selecting "All Teams" & "All Events"

- [ ] Empty State
  - Go to history page
  - Select filters that do not match to any events
  - Observe the empty state message shows up and you can reset back to "All Teams" & "All Events"

## Final checklist
